.github/PENDING_REVIEW_NOTIFICATION_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Hi {{ username }},
+
+It looks like you're assigned to this PR, but haven't taken any action for at least 2 days:
+{{ pr_list }}
+
+Please review the pending PRs as soon as possible, and reply to this thread once all the PRs are reviewed.
```

Comment:
Suggested change: Please review and deassign yourself from...

(This is because sometimes people forget to deassign themselves)